Scenarios/Monitoring/AVS-Dashboard/Bicep/AVSDashboard.bicep

```diff
@@ -336,3 +338,15 @@ resource Dashboard 'Microsoft.Portal/dashboards@2019-01-01-preview' = {
     'hidden-title': DashboardName
   }
 }
+
+resource Telemetry 'Microsoft.Resources/deployments@2021-04-01' = if (!TelemetryOptOut) {
+  name: 'pid-754599a0-0a6f-424a-b4c5-1b12be198ae8-${uniqueString(resourceGroup().id, Location)}'
```

Comment:
Can you include PrivateCloudResourceId in the uniqueString to match up with the other scenarios.

Reply:
Done Added. Thanks so much